.github/workflows/generate-dependencies.js

```diff
@@ -119,16 +128,15 @@ function writeFile(dir, result) {
     for (const [key, value] of Object.entries(components)) {
         const dir = value.directory;
         const provenanceFile = generateResult(key, value.version, dir);
-        const artifactRepos = provenanceFile['all-components']['artifact_repositories'];
+        const artifactRepos = provenanceFile['all_components']['component']['artifact_repositories'];
         value.dependencies.forEach(dependency => {
             artifactRepos.push({
                 content: 'binary',
                 protocol: 'HTTPS',
                 host: dependency.host,
-                path: dependency.path
+                paths: [dependency.path]
```

Comment:
Why change this from path to paths? Is this really a part of the new schema?

Reply:
This changes was needed due to the schema changes, https://gitlab.eng.vmware.com/provenance/provenance_data/-/blob/master/schema/provenance_data.schema-0.2.5-mvs.json